### PR TITLE
chore: homebrew typo

### PR DESCRIPTION
**Pull request by robinverduijn**

### CODE REVIEW

File: pypy/doc/install.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ extra packages. The choices for installing the base interpreter are:
 
 - Use conda (x86_64 windows, macOS, linux, arm64 linux)
 - Use your distribution package manager (linux)
-- Use homebew (macOS, but see issue 3697_)
+- Use homebrew (macOS, but see issue 3697_)
```
Comment (mattip) [inline pypy/doc/install.rst:9]:
```suggestion
- Use homebrew (macOS, but see issue 3696_)
```

**Comment by mattip** [comment]
Thanks for the typo fix

File: pypy/doc/install.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ extra packages. The choices for installing the base interpreter are:
 
 - Use conda (x86_64 windows, macOS, linux, arm64 linux)
 - Use your distribution package manager (linux)
-- Use homebew (macOS, but see issue 3697_)
+- Use homebrew (macOS, but see issue 3696_)
```
Comment (robinverduijn) [inline pypy/doc/install.rst:9]: By the way, I wonder if it's still necessary to mention issue 3696 -- as of May 23, 2024, there don't seem to be any problems installing pypy or pypy3 on MacOS Homebrew anymore.

Comment (mattip) [reply pypy/doc/install.rst:9]: thanks, done in 30ded988c20

**Pull request merged**